homeassistant/components/fitbit/const.py

```diff
@@ -0,0 +1,143 @@
+}
+DEFAULT_CLOCK_FORMAT: Final = "24H"
+
+FITBIT_RESOURCES_LIST: Final[dict[str, tuple[str, str | None, str | None]]] = {
+    "activities/activityCalories": ("Activity Calories", "cal", "fire"),
+    "activities/calories": ("Calories", "cal", "fire"),
+    "activities/caloriesBMR": ("Calories BMR", "cal", "fire"),
+    "activities/distance": ("Distance", "", "map-marker"),
+    "activities/elevation": ("Elevation", "", "walk"),
+    "activities/floors": ("Floors", "floors", "walk"),
+    "activities/heart": ("Resting Heart Rate", "bpm", "heart-pulse"),
+    "activities/minutesFairlyActive": ("Minutes Fairly Active", TIME_MINUTES, "walk"),
+    "activities/minutesLightlyActive": ("Minutes Lightly Active", TIME_MINUTES, "walk"),
+    "activities/minutesSedentary": (
+        "Minutes Sedentary",
+        TIME_MINUTES,
+        "seat-recline-normal",
+    ),
+    "activities/minutesVeryActive": ("Minutes Very Active", TIME_MINUTES, "run"),
+    "activities/steps": ("Steps", "steps", "walk"),
+    "activities/tracker/activityCalories": ("Tracker Activity Calories", "cal", "fire"),
+    "activities/tracker/calories": ("Tracker Calories", "cal", "fire"),
+    "activities/tracker/distance": ("Tracker Distance", "", "map-marker"),
+    "activities/tracker/elevation": ("Tracker Elevation", "", "walk"),
+    "activities/tracker/floors": ("Tracker Floors", "floors", "walk"),
+    "activities/tracker/minutesFairlyActive": (
+        "Tracker Minutes Fairly Active",
+        TIME_MINUTES,
+        "walk",
+    ),
+    "activities/tracker/minutesLightlyActive": (
+        "Tracker Minutes Lightly Active",
+        TIME_MINUTES,
+        "walk",
+    ),
+    "activities/tracker/minutesSedentary": (
+        "Tracker Minutes Sedentary",
+        TIME_MINUTES,
+        "seat-recline-normal",
+    ),
+    "activities/tracker/minutesVeryActive": (
+        "Tracker Minutes Very Active",
+        TIME_MINUTES,
+        "run",
+    ),
+    "activities/tracker/steps": ("Tracker Steps", "steps", "walk"),
+    "body/bmi": ("BMI", "BMI", "human"),
+    "body/fat": ("Body Fat", PERCENTAGE, "human"),
+    "body/weight": ("Weight", "", "human"),
+    "devices/battery": ("Battery", None, None),
+    "sleep/awakeningsCount": ("Awakenings Count", "times awaken", "sleep"),
+    "sleep/efficiency": ("Sleep Efficiency", PERCENTAGE, "sleep"),
+    "sleep/minutesAfterWakeup": ("Minutes After Wakeup", TIME_MINUTES, "sleep"),
+    "sleep/minutesAsleep": ("Sleep Minutes Asleep", TIME_MINUTES, "sleep"),
+    "sleep/minutesAwake": ("Sleep Minutes Awake", TIME_MINUTES, "sleep"),
+    "sleep/minutesToFallAsleep": (
+        "Sleep Minutes to Fall Asleep",
+        TIME_MINUTES,
+        "sleep",
+    ),
+    "sleep/startTime": ("Sleep Start Time", None, "clock"),
+    "sleep/timeInBed": ("Sleep Time in Bed", TIME_MINUTES, "hotel"),
+}
+
+FITBIT_MEASUREMENTS: Final[dict[str, dict[str, str]]] = {
+    "en_US": {
+        "duration": TIME_MILLISECONDS,
```

Comment:
I don't like these keys are being repeated many times as regular strings. Simple solution is to use constants, better solution is to use `TypedDict` even though all values are `str`.

Reply:
I would prefer the simple solution in this case :see_no_evil: